Doc/library/site.rst

```diff
@@ -187,7 +187,7 @@ Module contents
 
    Path to the base directory for the user site-packages.  Can be ``None`` if
    :func:`getuserbase` hasn't been called yet.  Default value is
-   :file:`~/.local` for UNIX and Mac OS X non-framework builds,
+   :file:`~/.local` for UNIX and macOS non-framework builds,
    :file:`~/Library/Python/{X.Y}` for Mac framework builds, and
```

Comment:
What is the correct name?

Reply:
I think this still applies to current macOS

Reply:
macOS?